### ISSUE TITLE
Align motion for Scannet test split?

### ISSUE THREAD

**Issue by siddharthKatageri** [Silverster98/HUMANISE#17]
Hi @Silverster98 

In the paper you mention that you follow Scannet's original train-test split and get 16.5k motions in 543 scenes for training and 3.1k motions in 100 scenes for testing.

1. How do you align motion for Scannet's test split?
In Scannet's test set, I see that every scene only has "*_00_vh_clean_2.ply" files and does not have other label files such as "*_00_vh_clean_2.labels.ply", ".aggregation.json" and "_vh_clean_2.0.010000.segs.json" which are needed for motion alignment.
Am I missing something?

2. Also I had a Minor doubt
Scannet has 707 scenes in its train split. Why does HUMANISE have 543 scene in its train split? Is it because while generating alignment motion it so happened that for some scenes, out of 10, none of the motions were selected as they could not follow the alignment constraints?

Looking forward for your reply.
Thank you!

**Comment by Silverster98** [comment]
> Hi @Silverster98
> 
> In the paper you mention that you follow Scannet's original train-test split and get 16.5k motions in 543 scenes for training and 3.1k motions in 100 scenes for testing.
> 
> 1. How do you align motion for Scannet's test split?
>    In Scannet's test set, I see that every scene only has "__00_vh_clean_2.ply" files and does not have other label files such as "__00_vh_clean_2.labels.ply", ".aggregation.json" and "_vh_clean_2.0.010000.segs.json" which are needed for motion alignment.
>    Am I missing something?
> 2. Also I had a Minor doubt
>    Scannet has 707 scenes in its train split. Why does HUMANISE have 543 scene in its train split? Is it because while generating alignment motion it so happened that for some scenes, out of 10, none of the motions were selected as they could not follow the alignment constraints?
> 
> Looking forward for your reply. Thank you!

1. Actually, we only use the training set of ScanNet for motion alignment, i.e., the scenes range from scene0000_00 to scene0706_00. We re-split the training set for training and testing. I am sorry for the incorrect statement causing this confusion.

2. Yes. Some scenes can not align with the selected motion sequences.

**Comment by siddharthKatageri** [comment]
Okay, I get it now. The split happens in the dataloader [here](https://github.com/Silverster98/HUMANISE/blob/11dee60fb8e76824bb55d1340036a5b739e3480b/model/dataloader.py#L105).

Thanks a lot for the clarification!
